templates/terraform/pre_update/spanner_instance_partition.go.tmpl

```diff
@@ -0,0 +1,19 @@
+fieldMask := []string{}
```

Comment:
This file is not needed as mmv1/templates/terraform/pre_update/spanner_instance_partition.go.tmpl is added.